ats/ats_jobs/models.py

```diff
@@ -0,0 +1,29 @@
+from django.db import models
+
+# Create your models here.
+
+class Category(models.Model):
+    title = models.CharField(max_length=128)
+    description = models.TextField()
+
+class Tag(models.Model):
+    title = models.CharField(max_length=128)
+    description = models.TextField()
+
+class JobTag(Tag):
+    AtsJob = models.ForeignKey('AtsJob', on_delete=models.CASCADE, related_name='ats_jobs')
+
+class AtsJob(models.Model):
```

Comment:
```suggestion
class Job(models.Model):
```
This is enough, you don't need to add `ATS` as a prefix